plugin/evm/vm_test.go

```diff
@@ -3173,3 +3176,142 @@ func TestStandaloneDB(t *testing.T) {
 	assert.False(t, isDBEmpty(vm.db))
 	assert.False(t, isDBEmpty(vm.acceptedBlockDB))
 }
+
+func TestValidatorState(t *testing.T) {
```

Comment:
Can we put this in a separate file eg, `vm_validator_state_test.go` as this file is already very long?

Reply:
It was in that file but intentionally moved to vm_test as this is the only test left in that package. But I agree vm_test became too long. Moving it back again.